recipe/README

```diff
@@ -1,26 +1,28 @@
-To build a conda pytorch package with GPU support
+# Package overview
 
-* Build the required Docker containers:
+Pytorch is configurable via a variety of environment variables, many of
+which are documented in the header of their setup.py. For example, the
+acceleration backend can be provided by CUDA, MKL, or OpenBLAS.
 
-    Dockerfiles can be found at: https://github.com/jjhelmus/docker-images
+Build.sh and bld.bat in this package mostly deal with setting the
+environment variables that we want to set, before calling setup.py, which
+is the entry point to the Pytorch build system.
 
-    CUDA 9.0: pkg_build_cos6_cuda90
-    CUDA 9.2: pkg_build_cos6_cuda92
-    CUDA 10.0: pkg_build_cos6_cuda100
 
-* Start the docker container using:
+# GPU building
 
-    ```
-    sudo nvidia-docker run -v `pwd`:/io -it pkg_build_cos6_cuda90
-    ```
+To build with GPU support, uncomment the line `#  - gpu                # [linux64 or win]`
```

Comment:
use '(linux and x86_64)' instead of 'linux64' please

Reply:
done